frontend/src/components/TouchableImage.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Text, Image, View,
+  TouchableHighlight,
+  Dimensions,
+} from 'react-native';
+
+export default function TouchableImage({
```

Comment:
I perfer this ways...everything defined in order and be exported at end.
what do you think?

```js
function TouchableImage() {...}

TouchableImage.propTypes = {...}

TouchableImage.defaultProps = {...}

export default TouchableImage;